src/sentry/grouping/utils.py

```diff
@@ -21,3 +22,11 @@ def get_rule_bool(value):
             return True
         elif value in ('0', 'no', 'false'):
             return False
+
+
+def resolve_fingerprint_values(values, event):
+    def get_fingerprint_value(value):
+        if value in TRANSACTION_FINGERPRINT_VALUES:
+            return event.data.get('transaction') or '<no-transaction>'
```

Comment:
Shouldn't we let this fall through if there is no transaction? That way, it would at least apply default fingerprinting if there's no transaction which might provide better results than grouping everything together.

Reply:
Let's not do this.